chains/43113/0x1741b9c475e0861a43b03f984928082ac4f3fb95/contract-information.json

```diff
@@ -0,0 +1,10 @@
+{
+  "address": "0x1741b9c475e0861a43b03f984928082ac4f3fb95",
+  "name": "ChainLink Token",
+  "symbol": "LINK.e",
+  "contractType": "ERC-20",
+  "decimals": 18,
+  "resourceLinks": [],
+  "logoUri": "https://raw.githubusercontent.com/ava-labs/subnet-assets/main/chains/43113/0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846/logo.png",
```

Comment:
the address on the logo needs to match this folder address

Reply:
updated